.stylelintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
+  "rules": {
+    "selector-class-pattern": [
```

Comment:
This rule is used to check if the class is using the BEM methodology. This is the only create rule because it is our core configuration.